app/assets/stylesheets/petitions/_forms.scss

```diff
@@ -31,11 +31,18 @@ select.form-control:hover {
   @extend %outdent-to-full-width;
 
   @include media(tablet) {
+    margin-left: -$gutter;
+    margin-right: -$gutter;
     padding-left: $gutter - 5px;
     padding-right: $gutter;
   }
 }
 
+.form-group + input[type=submit] {
```

Comment:
The problem is caused by the fact that `.form-group` is floated left in gov uk elements. Instead of removing the float from the `.form-group` (I suspect that's there for a reason) we could add a `clear: both;`. This seems to do the trick.